docs/topics/db/transactions.txt

```diff
@@ -290,45 +290,47 @@ Performing actions after commit
 
 Sometimes you need to perform an action related to the current database
 transaction, but only if the transaction successfully commits. Examples might
-include a `Celery`_ task, an email notification, or a cache invalidation.
+include a background task, an email notification, or a cache invalidation.
 
-.. _Celery: https://pypi.org/project/celery/
```

Comment:
There are many background task systems, especially more since when these docs were authored, I'm sure we should prioritize Celery over them.